net/ipfamily.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package net
+
+import (
+	"fmt"
+	"net"
+)
+
+// IPFamilyType refers to a specific family if not empty, i.e. "4" or "6".
+type IPFamilyType string
```

Comment:
I don't see the old type imported in k/k

```
$ grep -r netutils.IPFamily 
$ grep -r utilnet.IPFamily 
```

Reply:
I don't think we can do this.  We do claim a stable API.